PokeAlarm/Events/MonEvent.py

```diff
@@ -143,23 +143,19 @@ def generate_dts(self, locale, timezone, units):
             'encounter_id': self.enc_id,
             'mon_name': locale.get_pokemon_name(self.monster_id),
             'mon_id': self.monster_id,
-            'mon_id_3': "{:03}".format(self.monster_id),
-
-            # Time Remaining
-            'time_left': time[0],
-            '12h_time': time[1],
-            '24h_time': time[2],
-            'time_left_no_secs': time[3],
-            '12h_time_no_secs': time[4],
-            '24h_time_no_secs': time[5],
-            'time_left_raw_hours': time[6],
-            'time_left_raw_minutes': time[7],
-            'time_left_raw_seconds': time[8],
-
```

Comment:
Why were the above lines removed?
I think this is a standard part of dev now

Reply:
I accidentally pushed a preliminary commit, that's why I marked the whole PR as WIP. It's seriously broken now, but I'm super-busy atm and will push the full changeset later.

Reply:
As an outline, all the time fields get an `...or_empty` counterpart. Any given despawn time information is just an estimation if the despawn time is not verified, so the template configuration could reflect this.